packages/client/test/net/protocol/rlpxsender.spec.ts

```diff
@@ -52,14 +52,3 @@ describe('should receive message', async () => {
   })
   rlpxProtocol.events.emit('message', 1, 5)
 })
-
-describe('should catch errors', async () => {
-  const rlpxProtocol = { events: new EventEmitter() }
-  const sender = new RlpxSender(rlpxProtocol as Devp2pETH)
-  it('throws sendStatus error', () => {
-    assert.throws(() => sender.sendStatus({ id: 5 }), /not a function/, 'sendStatus error')
-  })
-  it('throws sendMessage error', () => {
-    assert.throws(() => sender.sendMessage(1, 5), /not a function/, 'sendMessage error')
```

Comment:
These events are deleted and do not seem related (?) to events, is this intended?

Reply:
Right, they aren't directly related but they're pointless tests and don't actually test anything other than that when you try to access a non-existent property on an object that Javascript throws.  I just stumbled across it because it referenced `EventEmitter`.